components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/mvkpert/shell_commands

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 
+# TODO: this was written as a DRAFT to mimick the decadal run
+# TODO: we obviously should change this to something else... 
+# TODO: maybe a standard F2010 case or something
+# TODO: probably doesn't really matter anyway, but still...
```

Comment:
I would remove this - it's not really a "to do" item, just a discussion, which I dont think is really relevent anyway